client/app/Streamfinder.jsx

```diff
@@ -104,7 +101,8 @@ class Streamfinder extends React.Component {
         </div>
       </Router>
     ) : (
-      <Home suggested={this.state.suggested} trending={this.state.trending} history={this.state.history} />
+     
+      <Home />
```

Comment:
Is this necessary? We will need Home not to render when users are not signed in. If you use / bookmark the virtual route `http://localhost:3000/Home` you won't have to click through anything else. We will need Home not to render for users who are not signed in. The reason this ternary was here was to facilitate SSR. As long the app is using SSR, this will cause bugs because you're asking to put Home into the DOM before there is a DOM (on the server, there is only HTML).